sdk-docs/project-structure.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Project Structure
+category: 61fcd8e1a448f5004215317c
+---
+
+> 🚧 Beta
+> 
+> This is a beta version and subject to change without notice. Pricing, terms, conditions and availability may change in the final version.
+
+# Project Structure
+The Trackunit SDK leverages the Nx build system to generate a mono-repo structure for app development. This structure allows developers to create and manage multiple apps and extensions within the same repository. By utilizing the mono-repo approach, developers can easily share code and resources between apps, reducing redundancy and enhancing maintainability. The shared codebase promotes consistency across applications and simplifies collaboration within teams. Therefore our suggested approach is to have *one* workspace setup and develop all your apps and extensions in it.
```

Comment:
Is the name Trackunit SDK, or Iris App SDK ? 🤔

Reply:
Iris App SDK

Reply:
Will fix